cmake/onnxruntime_unittests.cmake

```diff
@@ -595,6 +595,9 @@ endif()
 
 if(onnxruntime_USE_QNN)
   list(APPEND onnxruntime_test_framework_src_patterns  ${TEST_SRC_DIR}/providers/qnn/*)
+  list(APPEND onnxruntime_test_framework_src_patterns  ${TEST_SRC_DIR}/providers/qnn/conv/*)
```

Comment:
Shouldn't line 597 cover these all?

Reply:
Unfortunately, it doesn't glob recursively. I'll experiment with using `file(GLOB_RECURSE ...)` to see if we can avoid listing all directories manually.

Reply:
Actually, no need to create a folder for each op test to make it simple.

---
In reply to: [1142528421](https://github.com/microsoft/onnxruntime/pull/15126#discussion_r1142528421) [](ancestors = 1142528421)

Reply:
Done